tests/unit/api/metadata/test_v0.py

```diff
@@ -478,6 +478,17 @@ def test_popular_tables_success(self) -> None:
             self.assertEqual(response.status_code, HTTPStatus.OK)
             self.assertCountEqual(data.get('results'), self.expected_parsed_popular_tables)
 
+        with local_app.test_client() as test:
```

Comment:
The logic you've added for this PR add the `limit` to the popular tables get request, so these unit tests in the bigger scope of things don't actually unit test this logic since that logic is mocked. Meaning that these additions don't add any testing value, so I would prefer to remove them. What do you think?

Reply:
Yes, I agree. It didn't add much value. 

How do you do an integration test with the metadata API? I run local Docker compose to see if microservices work together, but I hope there is some kind of unit test for that.

Reply:
We currently don't have integration tests, and we still have the task of manually testing api changes.